assets/blocks/course-outline/lesson-block/edit.js

```diff
@@ -40,7 +40,16 @@ export const EditLessonBlock = ( props ) => {
 		insertBlocksAfter,
 	} = props;
 	const { selectNextBlock, removeBlock } = useDispatch( 'core/block-editor' );
-	const { setLessonStatus } = useDispatch( COURSE_STATUS_STORE );
+	const { setLessonStatus, trackLesson, ignoreLesson } = useDispatch(
+		COURSE_STATUS_STORE
+	);
+
+	// If the lesson has a title, add it to the tracked lessons in the status store.
```

Comment:
We could add it inside an useEffect to prevent call the actions every render. :)